webpack.config.js

```diff
@@ -110,34 +110,6 @@ const assets = {
   },
 };
 
-const tests = {
```

Comment:
Are these remove tests something we should be looking to replace? Or are they just being run already in a different manner?

Reply:
@kujenga The tests can be run in the terminal. (they're part of CI currently)